pkg/components/dex/component.go

```diff
@@ -245,19 +245,22 @@ type staticClient struct {
 }
 
 type component struct {
-	IngressHost          string         `hcl:"ingress_host,attr"`
-	IssuerHost           string         `hcl:"issuer_host,attr"`
-	Connectors           []connector    `hcl:"connector,block"`
-	StaticClients        []staticClient `hcl:"static_client,block"`
-	GSuiteJSONConfigPath string         `hcl:"gsuite_json_config_path,optional"`
+	IngressHost              string         `hcl:"ingress_host,attr"`
+	IssuerHost               string         `hcl:"issuer_host,attr"`
+	Connectors               []connector    `hcl:"connector,block"`
+	StaticClients            []staticClient `hcl:"static_client,block"`
+	GSuiteJSONConfigPath     string         `hcl:"gsuite_json_config_path,optional"`
+	CertManagerClusterIssuer string         `hcl:"certmanager_clusterissuer,optional"`
```

Comment:
Just a suggestion to use `certmanager_cluster_issuer`, in accordance with the annotation thats generally used ` cert-manager.io/cluster-issuer: nameOfClusterIssuer` ?

Reply:
Done.